drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/DropInConfigDataGenerator.kt

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2024 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 21/8/2024.
+ */
+
+package com.adyen.checkout.dropin.internal.ui
+
+import com.adyen.checkout.dropin.internal.ui.model.DropInParams
+
+internal class DropInConfigDataGenerator {
+
+    fun generate(configuration: DropInParams): Map<String, String> {
+        return mapOf(
+            "skipPaymentMethodList" to configuration.skipListWhenSinglePaymentMethod.toString(),
+            "openFirstStoredPaymentMethod" to configuration.showPreselectedStoredPaymentMethod.toString(),
```

Comment:
I think we can also send `configuration.isRemovingStoredPaymentMethodsEnabled`?

Reply:
The other platforms don't send this. Do you think we should add it anyway and align later?

Reply:
Yes, I think it is best to send what we can at this stage. Later others can send that too when they add the functionality.